rpa-builder/6.7/modules/ROOT/pages/toolbox-google-workspace-upload-to-google-drive.adoc

```diff
@@ -0,0 +1,64 @@
+= Upload to Google Drive 
+
+Upload files to your Google Drive. 
+
+You can export Google Docs, Sheets, and Slides into a selection of supported formats including PDF.
```

Comment:
This sentence can be removed, think it is still from the Download from Google Drive